framework_tvb/tvb/adapters/datatypes/db/removers/remover_connectivity.py

```diff
@@ -52,6 +52,11 @@ def remove_datatype(self, skip_validation=False):
             associated_stim = dao.get_generic_entity(StimuliRegionIndex, self.handled_datatype.gid, key)
             associated_mes = dao.get_generic_entity(ConnectivityMeasureIndex, self.handled_datatype.gid, key)
             associated_rvm = dao.get_generic_entity(RegionVolumeMappingIndex, self.handled_datatype.gid, key)
+            conn_measure_index_list = dao.get_generic_entity(ConnectivityMeasureIndex, self.handled_datatype.fk_connectivity_gid, key)
```

Comment:
I don't think this remover should be changed. The connectivity won't be removed as long as ConnMeasureIndex.handled_datatype (which will be a TSRegion in this case) exists. 
But instead this logic should go into the remover for RegionVolumeMappingIndex